contrib/gocryptfs/template.py

```diff
@@ -0,0 +1,37 @@
+pkgname = "gocryptfs"
+pkgver = "2.4.0"
+pkgrel = 0
+build_style = "go"
+make_build_args = [
+    "-ldflags="
+    + f" -X main.GitVersion={pkgver}"
+    + " -X main.GitVersionFuse=[vendored]",
+    ".",
+    "./gocryptfs-xray",
+]
+hostmakedepends = [
+    "go",
+    "pkgconf",
+]
+makedepends = [
+    "openssl-devel",
+]
+depends = [
+    "fuse",
+]
+pkgdesc = "Encrypted overlay filesystem"
+maintainer = "Nasado <hi@nasado.name>"
+license = "MIT"
+url = "https://github.com/rfjakob/gocryptfs"
```

Comment:
```suggestion
url = "https://nuetzlich.net/gocryptfs"
```

Reply:
I saw that website but by the looks of it it hasn't been updated in literal years, so I figured linking to the github would make more sense, but idk.

Reply:
ah, you are right